app/views/hyrax/dashboard/collections/_form.html.erb

```diff
@@ -1,34 +1,42 @@
-<%# OVERRIDE Hyrax 2.9.1 to make it say 'Default thumbnail' in the thumbnail select box instead of 'undefined' %>
+<%# OVERRIDE Hyrax v5.0.0rc2 to make it say 'Default thumbnail' in the thumbnail select box instead of 'undefined' %>
 <%= render "shared/nav_safety_modal" %>
-<div class="panel panel-default tabs" id="collection-edit-controls">
-  <ul class="nav nav-tabs" role="tablist">
-    <li class="active">
-      <a href="#description" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t('.tabs.description') %></a>
+<div class="tabs mt-4" id="collection-edit-controls">
+  <ul class="nav nav-tabs" id="dashboard-collection-tab" role="tablist">
+    <li class="nav-item">
+      <a href="#description" role="tab" data-toggle="tab" class="nav-link active nav-safety-confirm">
+        <%= t('.tabs.description') %>
+      </a>
     </li>
     <% if @form.persisted? %>
       <% if collection_brandable?(collection: @collection) %>
-      <li>
-        <a href="#branding" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t('.tabs.branding') %></a>
+      <li class="nav-item">
```

Comment:
Reading note: Hmm…based on past files, I would've thought this needed a Dom-id

Reply:
Another one copied from Hyrax, there seem to inconsistencies
https://github.com/samvera/hyrax/blob/double_combo/app/views/hyrax/dashboard/collections/_form.html.erb#L11